src/receive_imf.rs

```diff
@@ -2127,12 +2127,12 @@ async fn apply_group_changes(
         }
     } else if let Some(old_name) = mime_parser
         .get_header(HeaderDef::ChatGroupNameChanged)
-        // See create_or_lookup_group() for explanation
+        // See create_group() for explanation.
         .map(|s| s.trim())
     {
         if let Some(grpname) = mime_parser
             .get_header(HeaderDef::ChatGroupName)
-            // See create_or_lookup_group() for explanation
+            // See create_group() for explanation.
```

Comment:
Explanation what for, anyway? For why we trim() the group name?

Reply:
Exactly. There is a comment there that is longer than the corresponding code:
```
            // W/a for "Space added before long group names after MIME serialization/deserialization                                                                                                                 
            // #3650" issue. DC itself never creates group names with leading/trailing whitespace.
```
I'd prefer not to duplicate it

Reply:
Do you know what "W/a" means?

> I'd prefer not to duplicate it

Me too, we could remove it if we don't think it adds value. For me it wouldn't add value while reading at least because I neither would have wondered why we trim() - we trim() all the time - nor would I have understood it. But we can also just keep it if other people exist who did find it understandable.

Reply:
Removed these two referencing comments. Not that important indeed